sdk/storage/azure-storage-files-datalake/test/ut/recordings/DataLakeFileClientTest.AppendFileWithLease.json

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
I think recordings have been deleted from the repo. Can you confirm?

Reply:
Yeah, I am investigating test-proxy and will delete them soon.